src/ray/object_manager/test/create_request_queue_test.cc

```diff
@@ -262,19 +273,13 @@ TEST_F(CreateRequestQueueTest, TestEvictIfFull) {
 
 TEST(CreateRequestQueueParameterTest, TestNoEvictIfFull) {
   CreateRequestQueue queue(
-      /*max_retries=*/2,
       /*evict_if_full=*/false,
+      /*oom_grace_period_s=*/1,
       /*spill_object_callback=*/[&]() { return false; },
-      /*on_global_gc=*/[&]() {});
+      /*on_global_gc=*/[&]() {},
+      /*timer_callback=*/[&]() { return 0; });
 
-  bool first_try = true;
```

Comment:
I don't think we should remove this test code (it tests the lru_evict=True behavior).